core/math/sint/histogram.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sint
+
+import (
+	"math"
+	"sort"
+)
+
+type Histogram []int
```

Comment:
Please document all public types, functions and fields. These two links give a good guide on how to write godoc:

https://golang.org/doc/effective_go.html#commentary
https://blog.golang.org/godoc-documenting-go-code